install.sh

```diff
@@ -495,40 +506,17 @@ function configure_installation_variables() {
 	fi
 }
 
-function configure_gateway_node() {
-	local GATEWAY_LABEL="net.liqo.io/gateway=true"
-
-	# Check whether there is already a node labeled as gateway
-	GATEWAY=$(${KUBECTL} get node --selector "${GATEWAY_LABEL}" --output jsonpath="{.items[*].metadata.name}" 2>/dev/null) ||
-		fatal "[INSTALL] [CONFIGURE]" "Failed to detect whether a gateway node is already configured"
-
-	if [ "$(wc --words <<< "${GATEWAY}")" -ge 2 ]; then
-		fatal "[INSTALL] [CONFIGURE]" "More than one cluster node is labeled as gateway"
-	fi
-
-	if [ -z "${GATEWAY}" ]; then
-		# If not, select one node as gateway and label it accordingly
-		GATEWAY=$(${KUBECTL} get node --output jsonpath="{.items[-1].metadata.name}" 2>/dev/null) ||
-			fatal "[INSTALL] [CONFIGURE]" "Failed to retrieve a candidate gateway node"
-		${KUBECTL} label node "${GATEWAY}" "${GATEWAY_LABEL}" >/dev/null ||
-			fatal "[INSTALL] [CONFIGURE]" "Failed to label node ${GATEWAY} as gateway"
-	fi
-
-	GATEWAY_IP=$(${KUBECTL} get node "${GATEWAY}" -o jsonpath="{.status.addresses[0].address}" 2>/dev/null) ||
-		fatal "[INSTALL] [CONFIGURE]" "Failed to retrieve the IP address of the gateway node"
-	info "[INSTALL] [CONFIGURE]" "Gateway node: ${GATEWAY} (${GATEWAY_IP})"
-}
-
 function install_liqo() {
 	info "[INSTALL]" "Installing Liqo on your cluster..."
 
 	# Ignore errors that may occur if the namespace already exists
 	${KUBECTL} create namespace "${LIQO_NAMESPACE}" 1>/dev/null 2>&1 || true
-
 	local LIQO_CHART="${TMPDIR}/${LIQO_CHARTS_PATH}"
-	${HELM} dependency update "${LIQO_CHART}" >/dev/null ||
-		fatal "[INSTALL]" "Something went wrong while installing Liqo"
-	${HELM} install liqo --kube-context "${KUBECONFIG_CONTEXT}" --namespace "${LIQO_NAMESPACE}" "${LIQO_CHART}" \
+	info "[INSTALL]" "Packaging the chart..."
+	${HELM} package --version="${LIQO_CHART_VERSION}" --app-version="${LIQO_CHART_VERSION}" "${LIQO_CHART}" >/dev/null ||
+			fatal "[INSTALL]" "Something went wrong while installing Liqo"
+
+	${HELM} install liqo --kube-context "${KUBECONFIG_CONTEXT}" --namespace "${LIQO_NAMESPACE}" "liqo-${LIQO_CHART_VERSION}.tgz" \
 		--set tag="${LIQO_IMAGE_VERSION}" --set suffix="${LIQO_SUFFIX:-}" --set clusterName="${CLUSTER_NAME}" \
```

Comment:
I suppose `suffix` should be dropped, and `clusterName` dropped or changed to `discovery.config.clusterName`